userguide/system.rst

```diff
@@ -729,15 +729,34 @@ Self-Encrypting Drives
 
 %brand% version 11.1-U5 introduced Self-Encrypting Drive (SED) support.
 
-Three types of SED are supported:
+These SED specifications are supported:
 
 * Legacy interface for older ATA devices. **Not recommended for
   security-critical environments**
 
-* TCG OPAL 2 standard for newer consumer-grade devices (HDD or SSD over
-  PCIe or SATA)
+* `TCG OPAL 2 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Opal_SSC_v2.01_rev1.00.pdf>`__
+  standard for newer consumer-grade devices
 
-* TCG Enterprise standard for newer enterprise-grade SAS devices
+* `TCG Opalite <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Opalite_SSC_FAQ.pdf>`__
+  is a reduced form of OPAL 2
+
+* `TCG Pyrite Version 1 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Pyrite_SSC_v1.00_r1.00.pdf>`__
+  and
+  `TCG Pyrite Version 2 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Pyrite_SSC_v2.00_r1.00_PUB.pdf>`__
+  provide a logical equivalent of the legacy ATA security for non-ATA
+  devices. Hardware encryption is removed and only the drive firmware is
```

Comment:
I'd write here something like: "It is a version of Opalite with hardware encryption is removed ...".